### DISCUSSION TITLE
I think there are areas that need improvement

### DISCUSSION THREAD

**Discussion by 15953835029** [Comfy-Org/ComfyUI-Manager#1076] [Ideas]
1: If possible, please set up specifications for plug-in authors, such as the use of nodes, should be marked in the upper right corner of the node similar to "?" Click the icon to expand the usage description of the node, including the introduction of node options
2: When you use the openpose node node in controlnet, the page will be confused when you operate the pose pictures in the openpose node node

**Comment by ltdrdata** [comment]
1. This is an issue related to ComfyUI Core.
2. This is an issue related to the ControlNet Aux custom node.
